python/ray/tests/test_logging.py

```diff
@@ -332,12 +337,13 @@ def f():
         else:
             continue
         worker_id = re.match(pattern, str(path)).group(1)
+        worker_log_files.append(str(paths))
         ids.append(worker_id)
     counts = Counter(ids).values()
     for count in counts:
-        # There should be a "python-core-.*.log", "worker-.*.out",
-        # and "worker-.*.err"
-        assert count == 3
+        # For each worker, there should be a "python-core-.*.log", "worker-.*.out",
+        # and "worker-.*.err".
+        assert count % 3 == 0, worker_log_files
```

Comment:
```suggestion
        assert count % 3 == 0, worker_log_files
```

why `%` instead of `==`. I don't think you changed this behavior in this PR